### PR TITLE
fix(admin-ui): fetch pk from storage

### PR DESCRIPTION
**Pull request by frdomovic**
# fix(admin-ui): fetch pk from storage

### CODE REVIEW

File: node-ui/src/utils/displayFunctions.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const getDisplayPublicKey = (publicKey) => {
```
Comment (MatejVukosav) [inline node-ui/src/utils/displayFunctions.js]: I would call it `truncatePublicKey`. 
In the simplest version you can just pass publicKey but it is open to more granular modification by keeping the same name.

File: node-ui/src/utils/displayFunctions.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  return `
+      ${publicKey.split(":")[1].substring(0, 4)}...${publicKey
+    .split(":")[1]
+    .substring(
+      publicKey.split(":")[1].length - 4,
+      publicKey.split(":")[1].length
+    )}`;
+};
+
+export const getStatus = (active, revoked) => {
```
Comment (MatejVukosav) [inline node-ui/src/utils/displayFunctions.js:11]: I don't understand this function. Can `active` and `revoke` be true at the same time?

Comment (frdomovic) [reply node-ui/src/utils/displayFunctions.js:11]: No they can't, the other option i'm referring to in else is both false meaning -> exist in node and not revoked

File: node-ui/src/utils/displayFunctions.js (hunk 1 above)
Comment (frdomovic) [reply node-ui/src/utils/displayFunctions.js]: renamed

File: node-ui/src/utils/displayFunctions.js (hunk 2 above)
Comment (frdomovic) [reply node-ui/src/utils/displayFunctions.js:11]: I mean the component part could look like this
```
{revoked && <p>revoked</p>}
{active && <p>active</p>} 
```
but just minified code with `<div class=status()>status()</div>`

Comment (MatejVukosav) [reply node-ui/src/utils/displayFunctions.js:11]: Would this work?

```
if (active) {
    return "active";
  } else if (revoked) {
    return "revoked";
  } else {
    return "";
  }
```

Comment (frdomovic) [reply node-ui/src/utils/displayFunctions.js:11]: right, updated

**Pull request merged**